### PR TITLE
Add SQL based custom DC documentation; Update deployment setup

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: docs/custom_dc.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+-p 8080:8080 \
+-p 8081:8081 \
+-v $HOME/dc-data:/sqlite \
+gcr.io/datcom-ci/datacommons-website-compose:latest
+```
+
+If you have your own UI updates and build the docker image locally, replace the
+docker image with your locally one in the command.
+
+Now you can access a custom Data Commons site via
```
Comment (keyurva) [inline docs/custom_dc.md:93]: Perhaps also mention that the DC API will be on 8081 by default?

File: docs/custom_dc.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+restart the container.
+
+## Run in Cloud
+
+Custom Data Commons can be ran on the cloud as a production service. The SQLite
+approach can still be used. Note you need to copy the data folders into the
+cloud disk and mount it to the docker container then specify the enviornment
+variables based on the requirements of the cloud providers.
+
+We also provided a specific deployment setup on GCP that is based on Cloud SQL.
```
Comment (keyurva) [inline docs/custom_dc.md:108]: nit: "We _have_ also..."?

File: docs/custom_dc.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+Console](https://console.cloud.google.com/sql/instances). Set instance ID as
+`dc-graph`, choose the type as "MySQL" and create a database `datacommons`. You
+will need to set a user and password. Record the instance connection name in the
+form of "<project>:<region>:dc-graph"
+
+### Setup Google Cloud Storage
+
+Google Cloud Storage is used to hold the data CSV files. From the [Cloud
+Console](https://console.cloud.google.com/storage/browser), create a new bucket
+or pick up an existing bucket and upload the data CSV files there. It's
```
Comment (keyurva) [inline docs/custom_dc.md:122]: nit: s/pick up/pick

File: docs/custom_dc.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+recommended to create intermediate folders for the files for easier managment.
```
Comment (keyurva) [inline docs/custom_dc.md:123]: typo: management

File: docs/custom_dc.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+`gs://<bucket-name>/.../`.
+
+### Test Locally
+
+The CloudSQL backed custom DC can run locally. Need to run `gcloud auth
+application-default login` to get a credential json file locally. Then run:
+
+```bash
+docker run -it \
+-e MIXER_API_KEY=<YOUR_MIXER_API_KEY> \
```
Comment (keyurva) [inline docs/custom_dc.md:137]: `DC_API_KEY` here and elsewhere based on earlier comment.

**Review by keyurva — APPROVED** [review]
Thanks for the detailed documentation!

**Review by shifucun — COMMENTED** [review]
Thanks for the review!

**Pull request merged**